src/plugins/GUITestBase/src/GTUtilsDashboard.h

```diff
@@ -39,6 +39,28 @@ class GTUtilsDashboard {
                 Input,
                 ExternalTools };
 
+    // Dashboard notification class. Contains the type of notification (info/warning/error), the name of the workflow
+    // element to which the notification relates and the notification message
+    struct Notification {
+        enum class Type { INFO,
+                          WARN,
+                          ERR };
+
+        static const QString typesNames[3];
```

Comment:
Rename to typeNames or typeNameList. typesNames (when both words are plural) is not correct.